compiler/rustc_typeck/src/check/intrinsicck.rs

```diff
@@ -219,7 +221,7 @@ impl<'a, 'tcx> InlineAsmCtxt<'a, 'tcx> {
 
         // Check that the type implements Copy. The only case where this can
         // possibly fail is for SIMD types which don't #[derive(Copy)].
-        if !fcx.infcx.type_is_copy_modulo_regions(fcx.param_env, ty, DUMMY_SP) {
+        if !ty.is_copy_modulo_regions(self.tcx.at(expr.span), self.param_env) {
```

Comment:
Is there ever a possibility of `ty` having vars (region, ty, or const?) if so then this will ICE.

Reply:
o nvm, i think you handled that above